### DISCUSSION TITLE
I am not able to open Test Manager.

### DISCUSSION THREAD

**Discussion by akagupta98** [fzyzcjy/flutter_convenient_test#323] [Q&A]
I am always getting error -depends on integration_test from sdk which doesn't exist (the Flutter SDK is not available), version solving failed.

EVen though integration_test exist in my dependency and did flutter pub get also.

**Comment by fzyzcjy** [comment]
Well maybe show a bit more error logs etc, otherwise I also do not know what happens...

**Reply by akagupta98** [reply]
I am not getting any other errors. Just trying to run -
dart run convenient_test_manager_dart

and getting above error.

**Reply by fzyzcjy** [reply]
Maybe at least a bit more logs...

**Reply by akagupta98** [reply]
do we have any doc, how to setup manager with my tests? I dont see detailed doc and dont know how to configure it, that might be issue.

**Reply by fzyzcjy** [reply]
The readme does contain some introductions
